sway/auto_switch_kbd.py

```diff
@@ -1,16 +1,88 @@
 #! /usr/bin/python
+from typing import List
 
-from i3ipc import Connection, Event
+from i3ipc import Connection, Event, events
+
+KB_ID = "1:1:AT_Translated_Set_2_keyboard"
+MAX_WIN_HISTORY = 15
```

Comment:
It would be nice to have these parameters configurable via enviroment variables or script arguments

Reply:
Done